src/chrome/content/rules/E-rewards.com.xml

```diff
@@ -1,4 +1,7 @@
 <ruleset name="E-rewards.com">
-  <target host="www.e-rewards.com"/>
-  <rule from="^http://(?:www\.)?e-rewards\.com/" to="https://www.e-rewards.com/"/>
+	<target host="e-rewards.com"/>
+	<target host="www.e-rewards.com"/>
+
```

Comment:
flare.e-rewards.com

Reply:
Also https://beta.e-rewards.com

And https://pages.e-rewards.com fails with `not enough values to unpack (expected 2, got 1)`

And then there are lots of other hosts with errors of all types, including http://oa.e-rewards.com & http://owa.e-rewards.com which are cert errors as the cert is valid for `*.dynata.com, dynata.com`, but both of those are "Our website is currently unavailable.".

Would this be a reasonable time to use `*.e-rewards.com`

Reply:
Personally, I would rather whitelisting the ruleset for either coverage / fetch test instead of using a wildcard target to avoid tests. This is because there is a lot of different checking performed by Travis important for the ruleset quality. We also discourage wildcard targets unless the domain have a wildcard DNS record, to reduce the long term maintenance effort. 

If you would like to keep this ruleset as-is (without adding more hosts), I am ok to merge this given there was incomplete coverage. :smile:

Reply:
Given I have 500 odd files to fix, maintainers merging valid fixes rather than requiring I enhance them would be highly appreciated.

Note that using a wildcard was not to avoid tests.  It should be very obvious from my contributions that I am trying to get more testing to occur.  I would have accompanied a wildcard rule with tests for known working hosts.  However the state of this domain is clearly in flux, and/or deteriorating, so IMO it isnt great use of time to document all of the broken hosts which far outnumber the ones which are working.